src/components/select/select.tsx

```diff
@@ -0,0 +1,66 @@
+import React from "react"
+
+import * as SelectPrimitive from "@radix-ui/react-select"
+import clsx from "clsx"
+
+import s from "./select.module.scss"
+type Options = {
+  text: string
+  value: string
+}
+type Props = {
+  className?: string
+  onValueChange?: (items: string) => void
+  options: Options[]
+  placeholder?: string
+  value?: string
+} & React.ComponentPropsWithoutRef<typeof SelectPrimitive.Root>
+
+export const Select = React.forwardRef<React.ElementRef<typeof SelectPrimitive.Root>, Props>(
+  (
+    { children, className, disabled, onValueChange, options, placeholder, value, ...props },
```

Comment:
I suggest to use `...restProps` insted of `...props `
![image](https://github.com/incubatorLead/fotoFlow-ui-kit/assets/173442485/b34746d3-d323-4abf-b3f5-26f57604b74a)

Reply:
fixed